src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -25953,6 +25953,25 @@ void M6(C? c)
                 Diagnostic(ErrorCode.WRN_NullReferenceReceiver, "c").WithLocation(40, 9));
         }
 
+        [Fact]
+        [WorkItem(39868, "https://github.com/dotnet/roslyn/issues/39868")]
+        public void BadNullableDefaultArgument()
+        {
+            string source = @"
+public struct MyStruct
+{
+    static void M1(MyStruct? s = default(MyStruct)) { } // 1
```

Comment:
>MyStruct? [](start = 19, length = 9)

Please add a test for scenario when this is a not nullable reference type and the default value is bad.